eclipse-collections-code-generator/src/main/resources/api/set/primitiveSet.stg

```diff
@@ -44,6 +44,14 @@ public interface <name>Set extends <name>Iterable
      */
     <name>Set union(<name>Set set);
 
+    /**
+     * Returns the set of all objects that are members of both {@code this} and {@code set}. The intersection of
+     * [1, 2, 3] and [2, 3, 4] is the set [2, 3].
+     *
+     * @since 11.0.
+     */
+    <name>Set intersect(<name>Set set);
```

Comment:
should this have an override in `immutableSet.stg` for the return type?

Reply:
Do you mean `immutablePrimitiveSet.stg`? If yes, it is overridden there.